examples/game/lib/constraint.dart

```diff
@@ -41,3 +42,29 @@ class ConstraintRotationToMovement {
       node.rotation = _dampenRotation(node.rotation, target, dampening);
   }
 }
+
+class ConstraintRotationToNode {
+  ConstraintRotationToNode(this.targetNode, [this.baseRotation, this.dampening]);
```

Comment:
It's rare that you want optional arguments which are not keyword arguments.  Did you want {} instead of []?